llpc/lower/llpcSpirvLowerRayTracingIntrinsics.cpp

```diff
@@ -134,11 +134,15 @@ bool SpirvLowerRayTracingIntrinsics::processIntrinsicsFunction(Function *func) {
 // Create AmdExtD3DShaderIntrinsics_LoadDwordAtAddr, LoadDwordAtAddrx2, LoadDwordAtAddrx4,
 //
 // @param func : Function to create
-// @param loadPtrTy : Pointer type of the load value
-void SpirvLowerRayTracingIntrinsics::createLoadDwordAtAddr(Function *func, Type *loadPtrTy) {
+// @param loadTy : Base type of the load value
+// @param addressSpace : Address space of the memory to load the value from
+void SpirvLowerRayTracingIntrinsics::createLoadDwordAtAddr(Function *func, Type *loadTy,
+                                                           SPIRAddressSpace addressSpace) {
```

Comment:
I think we can omit the address space argument, the function always creates a pointer to the global addrspace.

Reply:
> I think we can omit the address space argument, the function always creates a pointer to the global addrspace.

Right now yes.